libtcmu_priv.h

```diff
@@ -55,6 +55,7 @@ struct tcmu_device {
 	unsigned int write_cache_enabled:1;
 	unsigned int solid_state_media:1;
 	unsigned int unmap_enabled:1;
+	unsigned int write_protect_enabled: 1;
```

Comment:
Please remove the white space before `1`.

Reply:
done.